lib/rules/array-callback-return.js

```diff
@@ -141,6 +141,8 @@ function isCallbackOfArrayMethod(node) {
 
 module.exports = {
     meta: {
+        type: "suggestion",
```

Comment:
This seems like it should be a `problem` since it's intended to point out bugs.

Reply:
Agree